src/pages/MainPage/CitiesSection.jsx

```diff
@@ -76,18 +75,21 @@ const CitiesSection = () => {
   ];
 
   return (
-    <section className="cities">
-      <div className="cities-container">
-        <h2 className="cities-title">Popular Cities</h2>
-        <div className="cities-info">
-          <h3 className="cities-subtitle">
+    <section className="min-h-[48.13rem] pt-28">
+      <div className="container mx-auto">
+        <h2 className="text-2rem font-fourth font-normal">Popular Cities</h2>
```

Comment:
`text-2rem`

Is it from overriding in the previous PR?

Reply:
I used some styles for the following sections, remembering that I had already added these styles in cfg.